sdk/inc/azure/core/az_mqtt5.h

```diff
@@ -318,7 +324,8 @@ enum az_event_type_mqtt5
   AZ_MQTT5_EVENT_SUBACK_RSP = _az_MAKE_EVENT(_az_FACILITY_CORE_MQTT5, 18),
 };
 
-// Porting 1. The following functions must be called by the implementation when data is received:
+// Porting az_mqtt 1. The following functions must be called by the implementation when data is
```

Comment:
We should extract this into a separate doxygen section.

Reply:
Not sure how that is meant to look, is this something like:

```
/**
* @brief Porting Guide
* 
* Porting 1: The following functions in az_mqtt5.h must be implemented 
* and will be called by the SDK to send data:
* #az_mqtt5_init, ... , #az_mqtt5_outbound_disconnect.
* 
* Porting 2: The following functions in az_mqtt5_property_bag.h must be ...
**/
```

Wouldn't it be better to have a readme documenting the porting process? It could live in sdk/docs/platform

Reply:
I don't like having documentation in 2 different places. I would prefer everything is in one place: doxygen.
We can also have a readme.md that points to the doxygen documentation.

Reply:
Also, you would need to generate the doxygen html and check to see if what you have looks good.

Reply:
I'll keep everything in doxygen, what I am still unsure about is what you mean by separate? Is it just to change the porting comments from `//` to `/** **/`? Or a doxygen comment like what I've written above which can be placed at the top of `az_mqtt.h`

Reply:
I was looking at the Doxygen docs: We could use either
1. @page https://www.doxygen.nl/manual/commands.html#cmdpage (and @subpage)
2. @module - https://www.doxygen.nl/manual/grouping.html

Reply:
Used @ page